### PR TITLE
Update entry count in home page #5159

### LINKED ISSUE #5159: Update entry count in home page
Update the entry count to 3 million as at 2016.03.30

### PR DESCRIPTION
**Pull request by yasirunilan**
Fixes #5159

### CODE REVIEW

File: src/main/webapp/js/index.js [hunk 1]
```diff
@@ -96,8 +96,8 @@ function submissionCounter(currentDate, baseDate, submissionPerHour, baseCount)
 onload = function() {
     
     // Parameters for the estimation calculation
-    var baseDate = new Date('October 30, 2015 00:00:00');  //The date the parameters were adjusted
-    var baseCount = 2000000;     //The submission count on the above date
+    var baseDate = new Date('March 30, 2016 00:00:00');  //The date the parameters were adjusted
+    var baseCount = 3000000;     //The submission count on the above date
     var submissionPerHour = 27; //The rate at which the submission count is growing
```
Comment (damithc) [inline src/main/webapp/js/index.js]: Update this to 128 too

Comment (yasirunilan) [inline src/main/webapp/js/index.js]: sorry.I didn't get that.Which one I need to update to 128??

Comment (damithc) [inline src/main/webapp/js/index.js]: 27--> 128 (line 101)

Comment (yasirunilan) [inline src/main/webapp/js/index.js]: updated

**Comment by wkurniawan07** [comment]
Looks good to merge!

**Pull request merged**